tests/functional/tests.py

```diff
@@ -102,6 +102,7 @@ class LogOutput(object):
 
```

Comment:
This comment is actually for the above line#101 and I think it introduced this line by mistake.
`class LogOutput(object):` is already at line#75 and hence this one needs to be removed.

Reply:
> This comment is actually for the above line#101 and I think it introduced this line by mistake.
> class LogOutput(object): is already at line#75 and hence this one needs to be removed.

yes @ebozduman, it is an old problem, I've just fixed now